tools/genwqe_echo.c

```diff
@@ -444,16 +443,14 @@ int main(int argc, char *argv[])
 		printf("\n");
 
  err_out:
-	if (!quiet) {
-		printf("--- UNIT #%x echo statistics ---\n"
-		       "%ld packets transmitted, %ld received, %ld lost, "
-		       "%ld%% packet loss, queue %lld usec\n",
-		       unit, packets_send,
-		       packets_received, (packets_send - packets_received),
-		       !packets_send ? 100 :
-		       100 * (packets_send - packets_received)/packets_send,
-		       wtime_usec);
-	}
+	printf("--- UNIT #%x echo statistics ---\n"
```

Comment:
No, please do not remove the if. The internal test cases require that _no_ output is generated. Just the exit code is checked.